alws/models.py

```diff
@@ -163,11 +163,13 @@ class BuildTask(Base):
         sqlalchemy.ForeignKey('build_task_refs.id'),
         nullable=False
     )
+    pgp_key_id = sqlalchemy.Column(sqlalchemy.VARCHAR(length=50), nullable=False)
     status = sqlalchemy.Column(sqlalchemy.Integer, nullable=False)
     index = sqlalchemy.Column(sqlalchemy.Integer, nullable=False)
     arch = sqlalchemy.Column(sqlalchemy.VARCHAR(length=50), nullable=False)
     ref = relationship('BuildTaskRef')
-    artifacts = relationship('BuildTaskArtifact')
+    artifacts = relationship('BuildTaskArtifact', backref='artifacts')
+    signed_artifacts = relationship("BuildTaskArtifact", backref='signed_artifacts')
```

Comment:
I think, it's better to move `pgp_key_id` to `Build` model, also, instead of `signed_artifacts` we can just replace links for unsigned one, since I doubt what we will store both of them.